### DISCUSSION TITLE
部分专有名词的修改的讨论

### DISCUSSION THREAD

**Discussion by NeilTohno** [Blinue/Magpie#522] [General]
> 你更改了一些专有名词：
> 
> scaling mode: 缩放模式->缩放方法
> Scaling configuration: 缩放配置->缩放设定
> Release notes: 发行说明->新版特性
> 
> 原本的表述和英文是一致的，你的改动是出于什么考虑？

不要直译这几个额，考虑一下国内软件的习惯、使用者的习惯。

缩放模式，模式这个词，样式，套路，特征，不合适,
scaling mode 这里指的是每一个可套用的缩放滤镜和效果，方法一词合适。
缩放配置，配置，和后面的配置文件冲突，
Scaling configuration，这里专指缩放相关的设定而已

_Originally posted by @NeilTohno in https://github.com/Blinue/Magpie/issues/518#issuecomment-1488220865_

发行说明，有点摸不着头脑，
国内软件用 “新版特性” 更多，更大众化一点

**Comment by Blinue** [comment]
总的来说，我希望避免口语化表达。

“发行说明“是很常见的：[Github](https://docs.github.com/zh/enterprise-server@3.5/repositories/releasing-projects-on-github/automatically-generated-release-notes)，[OneDrive](https://support.microsoft.com/zh-cn/office/onedrive-%E5%8F%91%E8%A1%8C%E8%AF%B4%E6%98%8E-845dcf18-f921-435e-bf28-4e24b95e5fc0)，[Debian](https://www.debian.org/releases/stable/releasenotes.zh-cn.html)，[维基百科](https://zh.wikipedia.org/zh-cn/%E7%99%BC%E4%BD%88%E7%89%88%E6%9C%AC%E9%80%9A%E7%9F%A5)

”设定“和”设置“、”配置“、”选项“等意思差不多，但在我见过的软件里很少出现。比如在 Windows 设置中搜索，”设置“、”配置“、”选项“都比较常见，而”设定“从未出现过。

至于”缩放模式“是从 v0.1 沿用至今的名词。最初想法是，使用何种算法放大到全屏有多种模式可供选择。”缩放方法“容易被理解成”使用何种方法实现缩放“而不是”使用何种算法进行缩放“。LS 里类似的选项名字叫”缩放类型“(Scaling Type)，我认为不太合适。

**Comment by NeilTohno** [comment]
> 总的来说，我希望避免口语化表达。
> 
> “发行说明“是很常见的：[Github](https://docs.github.com/zh/enterprise-server@3.5/repositories/releasing-projects-on-github/automatically-generated-release-notes)，[OneDrive](https://support.microsoft.com/zh-cn/office/onedrive-%E5%8F%91%E8%A1%8C%E8%AF%B4%E6%98%8E-845dcf18-f921-435e-bf28-4e24b95e5fc0)，[Debian](https://www.debian.org/releases/stable/releasenotes.zh-cn.html)，[维基百科](https://zh.wikipedia.org/zh-cn/%E7%99%BC%E4%BD%88%E7%89%88%E6%9C%AC%E9%80%9A%E7%9F%A5)
> 
> ”设定“和”设置“、”配置“、”选项“等意思差不多，但在我见过的软件里很少出现。比如在 Windows 设置中搜索，”设置“、”配置“、”选项“都比较常见，而”设定“从未出现过。
> 
> 至于”缩放模式“是从 v0.1 沿用至今的名词。最初想法是，使用何种算法放大到全屏有多种模式可供选择。”缩放方法“容易被理解成”使用何种方法实现缩放“而不是”使用何种算法进行缩放“。LS 里类似的选项名字叫”缩放类型“(Scaling Type)，我认为不太合适。

发行说明，只能被认为是国外软件发布的注释说明，不是合适的表达。
国内软件正式版出来了一般都是软件发布，而不是软件 “发行”。

设定，配置，选项，总是意思相近，所以才有必要和后面的导出设置文件区分；
软件设定，系统设定，日期设定是很常见的表达。

模式的英文翻译应为 pattern ；例词，线上模式，线下模式，教学模式，经营模式。
模式这个词，总是代表一直规律，经验，样板，模范一样的存在，因此不适合。
从 Magpie 这里来说，算法一词代表单一性，固定性，稳定性，非可组合的；
而方法一次更具有泛用性，可组合性，可变化性的。

以上。

**Comment by Blinue** [comment]
不同的人表达习惯有很大的不同，有争议的是正常的。改变这些专有名词需要更多人的意见。
我会暂时将这个 issue 转为 discussion。

**Comment by hooke007** [comment]
我认为项目的名词没有任何问题，我不是程序员出身，作为普通人中的一类，完全认可该软件的名词使用。

**Comment by debutow** [comment]
“配置”很常见意思也到位，不应改。

https://wikidiff.com/mode/pattern
pattern: (computing) A text string containing wildcards, used for matching.
mode: (computing) One of various related sets of rules for processing data.
综上mode比pattern合适。
“方法”我也认为会产生歧义，不如“模式”，或者直接叫“算法”得了（

Release notes ：“发布版本说明”或“版本说明” 如何？

**Comment by luojunyuan** [comment]
个人认为原来的表述没什么太大问题，平时使用中也逐渐习惯熟悉了。

也许简体中文里 "发行" -> "发布" 会更加通用一些。
"新版特性" 与 "Release notes" 相去甚远。
